content/spin/install.md

```diff
@@ -152,6 +155,27 @@ $ spin --help
 $ rustup update
 ```
 
+## Installing Default Templates and Plugins
+
+Spin has a variety of templates and plugins to make it easier to create Spin applications in your favorite programming language. 
+While the install script automatically installs these, if you used a different installation method, the following is a recommended post-installation step:
```

Comment:
Suggesting changing to

"While the install script automatically installs templates and plugins from the Spin repo, the JavaScript SDK, and Python SDKs, if you used a different installation method, the following is a recommended post-installation step:

Reply:
i modified it. Not word for word to your suggestion but still capturing what is being installed

Reply:
Thanks Kate.